db/migrate/20200713215815_add_criticality_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddCriticalityToProjects < ActiveRecord::Migration[6.0]
+  def change
+    add_column :projects, :criticality, :integer, :default => 1
```

Comment:
I'd leave the `default` out, unless you actually want to associate everything in the database with `1`. It might be easier and more visible to leave them `NULL`/unset until they're updated.